SpatialGDK/Extras/internal-documentation/release-process.md

```diff
@@ -174,21 +144,17 @@ Copy the latest release notes from `CHANGELOG.md` and paste them into the releas
 1. In `UnrealGDK`, merge `release` into `master`.
 
 **Documentation**
-1. Publish the docs to live using Improbadoc commands listed [here](https://improbableio.atlassian.net/wiki/spaces/GBU/pages/327485360/Publishing+GDK+Docs).
-1. Update the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1), moving the release from **Planned** to **Released**, and linking to the release.
```

Comment:
Any reason this was deleted along with the line below?

Reply:
We don't use these docs anymore

Reply:
Really? I thought Ernie kept the known issues on GitHub updated